### ISSUE TITLE
Allow support for multiple monitors

### ISSUE THREAD

**Issue by kristoferblack** [Nonary/ResolutionAutomation#1]
I run a headless HDMI dummy dongle on my machine for my Moonlight+Sunshine needs. Your script currently only acts upon the primary display (my main actual display) instead being able to specify my secondary monitor. At the very least, script should respect what display is specified in Sunshine options.

**Comment by Nonary** [comment]
It's something I've thought about doing as well but it is a limitation of the windows API call I am doing. I would have to make a lot of changes to support it.

I do have another automation that disables the primary display and configures the headless display as primary if you are interested in that as an alternative. 

I am planning to start developing for Sunshine and that is going to take a lot of my own personal time, before I get back to this script on this issue.

https://github.com/Nonary/MonitorSwapAutomation

**Comment by Nonary** [comment]
After reviewing it, I will not be supporting this feature because using Display Names in general is not a good idea for several reasons:

1. Disconnecting displays in Windows will cause Windows to reassign a new number, which often causes problems with Sunshine in general for users that try to use this feature.
2. Updates to display drivers will often cause the displays to change as well.
3. Even swapping inputs on the monitor, can also cause the display names to change.

In short, Sunshine will need to adjust its monitor selection feature because it is fundamentally flawed. Anyone who decides to use it, can and will run into breaking behaviors.
